sponsor/index.html

```diff
@@ -123,75 +132,242 @@ <h4>Sponsorship via Invoice</h4>
+      </li>
+      <li>Check the currency when donating (USD rates are provided by default). <br/>
+          <img src="img/paypal-donation.png" width="417"/>
       </li>
-      <li>Select a monthly donation sponsorship tier.
+      <li>Use the message field to support a specific activity: <br/>
+          <img src="img/paypal-message.png" width="417"/>
       </li>
       <li>
-         The OSGeo Treasurer will contact both you and the GeoServer PSC to acknowledge your sponsorship:
+         The OSGeo Treasurer will contact both you and the GeoServer PSC to acknowledge your sponsorship.
          <ul>
-         <li>Please have a logo ready for your organization if you wish to be recognized publicly.</li>
-         <li>As with any OSGeo activity your sponsorship (at your request) will be cross listed on the OSGeo sponsorship page.</li>
+         <li>Donations: We will ask individuals for their name, and a useful link.</li>
+         <li>Sponsorship: We will ask organisations for their logo to be listed on the GeoServer and OSGeo website.</li>
          </ul>
       </li>
-      <li>
-        Example email requesting listing on the OSGeo sponsorship page.
-        <blockquote>
-          <p>Subject: setting up geoserver sponsorship</p>
-          <p>To: treasurer@osgeo.org</p>
-
-          <p>Treasurer:</p>
-          <p>
-          I have used GitHub Sponsors page to setup a monthly donation of $50 to the GeoServer project.</p>
-          <p>
-          Please observe the attached logo for OSGeo Bronze sponsorship.</p>
-     
-          <p>Anaximander<br/>
-     
-          <p>attachement: <i>milesian.svg</i></p>
-        </blockquote>
-      </li>
     </ol>
+    
 </section>
+
+<section id="recommendations">
+    <h2>Sponsorship Recognition and Recommendations</h2>
+    <p class="lead">
+      Financial support indicates a commitment to the sustainability of the GeoServer, for those that depend on GeoServer's availability. Sponsorship is not a replacement for commercial support.
+    </p>
+    <p>
+      We appreciate all levels of support, the following recommend donation levels provide the team an opportunity to acknowledge your financial support.
+    </p>
+    <h4>Individuals</h4>
+    <p>
+       Donations help the project steering committee support developers in small ways.
+       Examples include covering travel expenses and code-sprint attendance.
+    </p>
+    <ul>
+      <li>$5/monthly Enthusiast:Fans of OSGeo, badge your repository as supporting OSGeo with a small donation.</li>
+      <li>$10/monthly Supporter: GIS Professionals, name listed as 2024 Donations on GeoServer website.</li>
+      <li>$50/monthly Bronze Sponsor: Individual consultants, Logo listed as 2024 Sponsor on <a href="https://www.osgeo.org/sponsors/">GeoServer website</a>, cross-listed to OSGeo website as a bronze sponsors, bronze sponsor logo, be sure to setup your <a href="http://osgeo.org/service-providers/">service provider page</a>.</li>
+    </ul>
```

Comment:
You go on talking about OSGeo, but most GeoServer users I know have no idea what OSGeo is to start with.
I'd add a statement clarifying the relationship and indicate that with a big enough donation, one gets recognized also as an OSGeo sponsor, as a bonus.

Reply:
Except sometimes we do fundraising for a specific goal; the point is they do not get to choose the goal :)

Reply:
I thought I did that at the top of the page, let me try and address it here also.

Reply:
<img width="809" alt="image" src="https://github.com/geoserver/geoserver.github.io/assets/629681/609feec3-3126-4218-b79c-fc5a54c0d445">